osm2lanes-web/src/main.rs

```diff
@@ -27,6 +30,27 @@ use map::MapComponent;
 #[global_allocator]
 static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 
+#[derive(Properties, PartialEq)]
+pub struct CodeProps {
+    pub code: String,
+}
+
+#[function_component(CodeHtml)]
+pub fn code_html(props: &CodeProps) -> Html {
+    let html = {
+        let ss = SyntaxSet::load_defaults_newlines();
+        let ts = ThemeSet::load_defaults();
+        let syntax = ss
+            .find_syntax_by_token("json")
+            .unwrap_or_else(|| ss.find_syntax_plain_text());
+        highlighted_html_for_string(&props.code, &ss, syntax, &ts.themes["base16-ocean.dark"])
+            .unwrap()
+    };
+    let div = gloo_utils::document().create_element("code").unwrap();
```

Comment:
Hmm, should we limit the max height and use `overflow: scroll` or something like that?